docs/user/openstack/external_load_balancer.md

```diff
@@ -0,0 +1,225 @@
+# Deploying OpenShift with an user managed load balancer
+
+This document explains how to deploy OpenShift with an user managed load balancer, rather than with the OpenShift managed (that is based on HAproxy & Keepalived).
+
+## Table of Contents
+- [Deploying OpenShift with an user managed load balancer](#deploying-openshift-with-an-user-managed-load-balancer)
+  - [Table of Contents](#table-of-contents)
+  - [Common prerequisites](#common-prerequisites)
+  - [Deploy the load-balancer](#deploy-the-load-balancer)
+  - [Deploy OpenShift](#deploy-openshift)
+  - [Known limitations](#known-limitations)
+  - [Notes](#notes)
+
+## Common prerequisites
+
+* When deploying OpenShift with an user managed load balancer, it's required to bring your own network(s) before
+  the deployment. This can be a tenant network or a provider network.
+* The load balancer(s) will have to be deployed before installing OpenShift.
+  * It has be connected to the network(s) where OpenShift will be deployed.
+  * If it's a server managed by OpenStack, allowed address pairs have to be configured
+    for the port that will serve API and Ingress traffic, otherwise the VIP traffic will be rejected by the
+    OpenStack SDN.
+  * The firewall has to allow the following traffic on the load balancer (if a server managed by OpenStack, create a security group):
+    * 22/TCP - SSH (to allow Ansible to perform remote tasks from the host where it runs). This rule can be removed after the load balancer is deployed.
+    * 6443/TCP (from within and outside the cluster) - OpenShift API
+    * 80/TCP (from within and outside the cluster) - Ingress HTTP
+    * 443/TCP (from within and outside the cluster) - Ingress HTTPS
+    * 22623/TCP (from within the OCP network) - Machine Config Server
+
+## Deploy the load balancer
+
+Before you install OpenShift, you must provision at least one load balancer.
+The load balancer will manage the VIPs for API, Ingress and Machine Config Server services.
+If you deploy in production, at least two load-balancers would be deployed per network fabric for high availability.
+
+You can use your own solution that suits your needs, or you can use this [Ansible role](https://github.com/shiftstack/ansible-role-routed-lb)
+that has been created for testing purpose and can be used as an example. This role has not been tested in production,
+therefore we can't recommand to use it outside of testing environments.
+
+In our example, we'll deploy the OpenShift cluster on one subnet (192.168.10.0/24). The API VIP will be 192.168.10.5 and the Ingress VIP will be 192.168.10.7.
```

Comment:
Personally, I would move everything from this paragraph to the end of the section into the https://github.com/shiftstack/ansible-role-routed-lb repo as an example, and simply link to it from here. It's very much specific to that role, right?